src/commands/install.yml

```diff
@@ -20,15 +20,14 @@ steps:
       command: |
 
         # If the version is not explicitly specified then use the existing one if possible.
-        if [ "<< parameters.version >>" = "latest" ]  && command -v secrethub >/dev/null 2>&1; then
+        if [ "<< parameters.version >>" = "latest" ] && command -v secrethub >/dev/null 2>&1; then
             exit 0
         fi
 
         # Get the latest version.
         VERSION="<< parameters.version >>"
         if [ "$VERSION" = "latest" ]; then
-            VERSION=$(curl --silent --show-error --location --fail --retry 3 \
```

Comment:
Where did all these options go? `--show-error` `--retry 3` etc. ?

Reply:
They're there: `-sSfL`. Only the `retry 3` is not there, do we want that?

Reply:
I don't see a reason to change it now.